awx/playbooks/project_update.yml

```diff
@@ -148,18 +149,12 @@
         register: doesRequirementsExist
 
       - name: fetch galaxy roles from requirements.yml
-        command: ansible-galaxy install -r requirements.yml -p {{project_path|quote}}/roles/
+        command: ansible-galaxy install -r requirements.yml -p {{roles_destination|quote}}
```

Comment:
Probably should figure out how to incorporate https://github.com/ansible/awx/pull/3887

Reply:
That proposal has a lot of unanswered questions. Having multiple folders under `roles/` seems like it wouldn't work right. I get the playbook-adjacent proposal, some variation of that might work.

If we do some variation of that request, I wouldn't anticipate much more trouble integrating it with this than with what we have now. It would be messy, involve searching around a couple places.

Reply:
I certainly wouldn't be in love with keeping the requirements files in the same directory subtree that roles are getting installed to.